modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/dag/schema.scala

```diff
@@ -11,6 +11,24 @@ object schema {
 
   case class GenericMetadata()
 
+  @derive(eqv, show)
+  sealed trait SignatureType extends EnumEntry
+
+  object SignatureType extends Enum[SignatureType] with SignatureTypeCodecs {
+    case object Ecdsa extends SignatureType
+    case object Ecdsa_recovery extends SignatureType
+    case object Ed25519 extends SignatureType
+    case object Schnorr_1 extends SignatureType
+    case object Schnorr_poseidon extends SignatureType
+
+    override def values: IndexedSeq[SignatureType] = findValues
+  }
+
+  trait SignatureTypeCodecs {
+    implicit val encode: Encoder[SignatureType] =
```

Comment:
Looks copy-pasted from another place. The name of the field should match the content. What's more, "codecs" means both encoder and decoder, but here is the only one. I saw the same in the previous pull request. Could you fix both (see how it's done for node state)

Reply:
`SignatureType` is a duplicate of `ChainObjectStatus` from the same file. Both Codecs traits have been renamed `*Namer` and the field is `name`.

Reply:
The field name is unchanged

Reply:
I get what you are trying to say now. Changed to match NodeState.